### DISCUSSION TITLE
The column should have max length to avoid collapse layout

### DISCUSSION THREAD

**Discussion by zhenyuan0502** [minhhungit/ConsoleTableExt#27] [Ideas]
As title, the column should be adjusted from the current view length if it's possible, otherwise, define an MaxLength to replace the outer limit by `...` or allowing to break new line should be considered.

**Comment by minhhungit** [comment]
@zhenyuan0502  At this time you can use [WithFormatter](https://github.com/minhhungit/ConsoleTableExt/discussions/14#discussioncomment-352757) to limit length of text

I will add maxlenght method later

**Comment by minhhungit** [comment]
Will convert this to discussion
